src/workloads/execution/EventCollection.yml

```diff
@@ -0,0 +1,71 @@
+SchemaVersion: 2018-07-01
+Owner: Storage Execution
+Description: |
+  Inserts many documents into an event collection.
+
+GlobalDefaults:
+  Collection: &Collection Collection0
+  Database: &db test
+
+Actors:
+  - Name: CreateEventCollection
+    Type: RunCommand
+    Phases:
+      - Repeat: 1
+        Database: *db
+        Operation:
+          OperationName: RunCommand
+          OperationCommand:
+            drop: *Collection
+      - Repeat: 1
+        Database: *db
+        Operation:
+          OperationName: RunCommand
+          OperationCommand:
+            create: *Collection
+            clusteredIndex:
+              { key: { _id: 1 }, unique: true, autoAssign: "timestamp" }
+      - &Nop { Nop: true }
+  - Name: Insert
+    Type: Loader
+    Database: *db
+    Threads: 1
+    Phases:
+      - *Nop
+      - *Nop
+      - Repeat: 1
+        Database: *db
+        Threads: 1
+        CollectionCount: 1
+        DocumentCount: 1000000
+        BatchSize: 1000
+        Document:
```

Comment:
How many bytes does each document end up being?

Reply:
Each document, with field names + values, should be 156 byes (assuming there is no overhead)